src/Evolution/Systems/RelativisticEuler/Valencia/System.hpp

```diff
@@ -40,6 +40,7 @@ struct System {
       tmpl::list<Tags::TildeD, Tags::TildeTau, Tags::TildeS<Dim>>>;
   using flux_variables =
       tmpl::list<Tags::TildeD, Tags::TildeTau, Tags::TildeS<Dim>>;
+  using non_conservative_variables = tmpl::list<>;
```

Comment:
why doesn't Burgers need this?

Reply:
Burgers has `has_primitive_and_conserved_vars = false`, so it doesn't take the constexpr branch that makes a distinction between the conservative and non-conservative variables in `SetVariables`.

I could add it to Burgers anyways, though, if it seems like the interface should be consistent among systems that are in flux conservative form.